stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.12.1
+version: 7.13.0
```

Comment:
because this is a minor version bump, you still need to do the following (look at previous commits to see an example):
1. add a change-log to `UPDATE.md`
1. link that change-log in `README.md`